examples/l3out_floating_svi/main.tf

```diff
@@ -30,7 +30,16 @@ resource "aci_logical_node_profile" "node_profile" {
 
 resource "aci_logical_interface_profile" "interface_profile" {
   logical_node_profile_dn = aci_logical_node_profile.node_profile.id
-  name                    = "demo_int_prof"
+  name = "demo_int_prof"
```

Comment:
Can you run `terraform fmt` on the example files?

Reply:
ok